client/components/Facilitator/Components/Cohorts/index.jsx

```diff
@@ -0,0 +1,279 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import {
+    Button,
+    Card,
+    Container,
+    Form,
+    Grid,
+    Header,
+    Icon,
+    Input,
+    Loader,
+    Menu,
+    Modal
+} from 'semantic-ui-react';
+import {
+    getCohorts,
+    getCohort,
+    setCohort,
+    createCohort
+} from '@client/actions/cohort';
+import { initialState } from '@client/reducers/cohort';
```

Comment:
This component is where a new cohort will be created from user input, then saved to the DB. I guess I don't have to use this, but I thought it would be useful for consistency 🤷‍♂ I can remove it.